layouts/partials/footer/_footer.scss

```diff
@@ -7,58 +7,60 @@
     padding-top: $spacing-12;
     padding-bottom: $spacing-12;
   }
-}
 
-.footer .row {
-  @media ($large-up) {
-    display: flex;
-    align-items: flex-end;
-    margin-left: -$spacing-5;
-    margin-right: -$spacing-5;
-  }
-}
+  .row {
+    @media ($large-up) {
+      display: flex;
+      align-items: flex-end;
+      margin-left: -$spacing-5;
+      margin-right: -$spacing-5;
+    }
 
-.footer .row .col {
-  @media ($large-up) {
-    width: 25%;
-    margin-left: $spacing-5;
-    margin-right: $spacing-5;
+    .col {
+      @media ($large-up) {
+        width: 25%;
+        margin-left: $spacing-5;
+        margin-right: $spacing-5;
+      }
+    }
   }
-}
 
-.col + .col {
-  margin-top: $spacing-9;
+  .body li {
+    a {
+      color: $c-dark-high-emphasis;
+      text-decoration: none;
 
-  @media ($medium-up) {
-    margin-top: $spacing-12;
+      &:hover {
+        text-decoration: underline;
+      }
+    }
+
+    & + li {
+      margin-top: $spacing-5;
+    }
   }
 
-  @media ($large-up) {
-    margin-top: 0;
+  .col .body li.last {
+    text-align: left;
+
+    @media ($large-up) {
+      text-align: right;
+    }
   }
-}
 
-.footer .row .col #logo {
-  margin-bottom: $spacing-12;
-}
+  .col + .col {
+    margin-top: $spacing-9;
 
-.col ul li.last {
-  text-align: left;
+    @media ($medium-up) {
+      margin-top: $spacing-12;
+    }
 
-  @media ($large-up) {
-    text-align: right;
+    @media ($large-up) {
+      margin-top: 0;
+    }
   }
 }
 
-ul.body-1 > li + li {
-  margin-top: $spacing-5;
-}
-
-ul.body-1 li a {
-  color: $c-dark-high-emphasis;
-  text-decoration: none;
+#logo {
```

Comment:
Try not to use ids as CSS selectors (specificity gets increased and they're not reusable).